lib/galaxy/tool_shed/util/repository_util.py

```diff
@@ -208,15 +208,17 @@ def get_installed_repository(app, tool_shed=None, name=None, owner=None, changes
     """
```

Comment:
Should we remove `refresh=False` from the function arguments a couple of lines above?